modules/apps/commerce/headless/headless-commerce/headless-commerce-admin-inventory-impl/src/main/java/com/liferay/headless/commerce/admin/inventory/internal/dto/v1_0/WarehouseDTOConverter.java

```diff
@@ -48,8 +50,11 @@ public Warehouse toDTO(DTOConverterContext dtoConverterContext)
 			_commerceInventoryWarehouseService.getCommerceInventoryWarehouse(
 				(Long)dtoConverterContext.getId());
 
+		Locale locale = dtoConverterContext.getLocale();
```

Comment:
you set this locale but I don't see where it is used

Reply:
I've removed this locale